### PR TITLE
[OTE][MPA] Apply ignored_labels to C-IL for semantic segmentation

### PR DESCRIPTION
**Pull request by chuneuny-emily**
### This PR includes :
- Add 'ignored_labels' for each item in the OTEDataset used for the semantic segmentation task
- MPA task modifies 'meta_keys' (in 'Collect') so that 'ignored_labels' are passed through 'img_metas'

### CODE REVIEW

**Comment by chuneuny-emily** [comment]
run ote-test

File: external/mmsegmentation/segmentation_tasks/extension/datasets/mmdataset.py [hunk 1]
```diff
@@ -95,15 +95,18 @@ def __getitem__(self, index):
             :return data_info: dictionary that contains the image and image metadata, as well as the labels of
             the objects in the image
             """
-
             dataset = self.ote_dataset
             item = dataset[index]
 
+            label_idx = {label.id: i for i, label in enumerate(self.labels)}
```
Comment (goodsong81) [inline external/mmsegmentation/segmentation_tasks/extension/datasets/mmdataset.py]: No need to make this id->idx table for each item; please refer to detection PR.

File: external/model-preparation-algorithm/mpa_tasks/apis/segmentation/task.py [hunk 2]
```diff
@@ -277,6 +277,17 @@ def patch_color_conversion(pipeline):
...
+                    if pipeline_step.get('meta_keys', False):
+                        meta_keys = list(pipeline_step['meta_keys'])
+                        meta_keys.append('ignored_labels')
+                    else:
+                        meta_keys = (
+                            'filename', 'ori_filename', 'ori_shape',
+                            'img_shape', 'pad_shape', 'scale_factor', 'flip',
+                            'flip_direction', 'img_norm_cfg', 'ignored_labels'
+                        )
+                    pipeline_step['meta_keys'] = set(meta_keys)
```
Comment (goodsong81) [inline external/model-preparation-algorithm/mpa_tasks/apis/segmentation/task.py]: Discuss w/ @harimkang for better solution.

**Review by goodsong81 — CHANGES_REQUESTED** [review]

File: external/mmsegmentation/segmentation_tasks/extension/datasets/mmdataset.py (hunk 1 above)
Comment (chuneuny-emily) [reply external/mmsegmentation/segmentation_tasks/extension/datasets/mmdataset.py]: Moved to init().

**Review by harimkang — APPROVED** [review]
LGTM

**Review by goodsong81 — APPROVED** [review]
LGTM.

**Pull request merged**